examples/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: run
+run:
+	mkdir -p log
+	chmod 777 log
+	docker-compose build
+	docker-compose up -d srv
+	sleep 10 # wait for bootstrap
```

Comment:
Instead of a direct sleep could we instead use some docker or docker-compose command to check if everything is ready?

Reply:
I was considering it. We could try e.g.  `docker compose ps --status=running`. However, I think that still it can say that it is "up" to early. I do not want to overcomplicate this example.